src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_config.json

```diff
@@ -5,14 +5,18 @@
     "value": {
       "AllowedValues": [
         "AWS::ACM::Certificate",
+        "AWS::ApiGateway::DomainName",
+        "AWS::ApiGateway::Method",
         "AWS::ApiGateway::RestApi",
         "AWS::ApiGateway::Stage",
+        "AWS::ApiGatewayV2::DomainName",
```

Comment:
[not seeing these `AWS::ApiGateway` additions documented here](https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#amazonapigateway) or in the [Config console](https://console.aws.amazon.com/config/home#/first-run/configuration)

Do you know if they are recent additions or if there is somewhere else to find them?

<img width="652" alt="Screen Shot 2020-04-24 at 3 57 11 PM" src="https://user-images.githubusercontent.com/7014355/80263085-54fd3580-8644-11ea-875a-d193cd6168a7.png">

Reply:
🤔 Huh! I swear they were in here last week.